test/PrintAsObjC/classes.swift

```diff
@@ -318,6 +318,13 @@ typealias AliasForNSRect = NSRect
   func testBridgingOptionality(_ a: UnsafePointer<Int>?, b: UnsafeMutablePointer<Int>!, c: AutoreleasingUnsafeMutablePointer<Methods?>?) {}
 }
 
+// CHECK-LABEL: IB_DESIGNABLE
+// CHECK-LABEL: @interface MyDesignableObject : NSObject
```

Comment:
Are you sure of that? I'm using `CHECK LABEL` because using `CHECK NEXT` was failing on the mangled name. Or at least is what I've understood from the test report.